spec/factories/ont/instruments.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :ont_instrument, class: 'Ont::Instrument' do
+    sequence(:name) { |n| "O#{n}" }
+    instrument_type { 'GridION' }
+    max_number_of_flowcells { 5 }
+
+    factory :ont_gridion do
```

Comment:
Is this extra factory needed if you change the base factory to gridion?

Reply:
ont_instrument is a generic one. We could remove 'GridION' from that but instrument_type uses enum values.
ont_gridion is more specific and definitively with 5 max number of flowcells.
We are using both in tests.